terraform/aws/analytical-platform-management-production/terraform-state/data.tf

```diff
@@ -7,3 +7,38 @@ data "aws_iam_session_context" "session" {
 
   arn = data.aws_caller_identity.session.arn
```

Comment:
Is the PR to change the assume role policy for the CI role coming up in another PR or are they getting a role of their own?

Reply:
They are using our repository so AFAIK that doesn't need changing